app/controllers/dashboard/protocols_controller.rb

```diff
@@ -97,6 +97,8 @@ def new
     session[:protocol_type] = params[:protocol_type]
     gon.rm_id_api_url = RESEARCH_MASTER_API
     gon.rm_id_api_token = RMID_API_TOKEN
+    @rmid_server_down = @protocol.rmid_server_status
```

Comment:
Since this is repeated multiple times, it might be worth putting inside a helper method in Dashboard::BaseController